core/templates/pages/contributor-dashboard-admin-page/contributor-dashboard-admin-summary.model.ts

```diff
@@ -61,6 +71,15 @@ export class TranslationSubmitterStats {
       summaryDict.last_contributed_in_days
     );
   }
+
+  getContributorAttributes(
+    contributorStats: ContributorStats,
+    formatContributorAttributesService: FormatContributorAttributesService
```

Comment:
Just wondering, why do you need to pass this in, rather than import it at the top? 

Ditto below.

Reply:
Done